### PR TITLE
TabManager: Add Unload checked tabs action

### PR DESCRIPTION
**Pull request by SGOrava**

### CODE REVIEW

File: src/plugins/TabManager/tabmanagerwidget.cpp [hunk 1]
```diff
@@ -471,7 +472,7 @@ void TabManagerWidget::processActions()
                 continue;
             }
 
-            if (command == "closeSelection") {
+            if (command == "closeSelection" || command == "unloadSelection") {
```
Comment (nowrep) [inline src/plugins/TabManager/tabmanagerwidget.cpp]: Umm yeah, this is `if (A) { action1() } else if (B) { action1(); }` :D @srazi

Comment (SGOrava) [reply src/plugins/TabManager/tabmanagerwidget.cpp]: It is strange construction, but for this small update i did not want to change a lot of things...

Comment (srazi) [reply src/plugins/TabManager/tabmanagerwidget.cpp]: Sometimes we code and sometimes we do magic :D

I'll fix it, ASAP.

Comment (srazi) [reply src/plugins/TabManager/tabmanagerwidget.cpp]: :D ea0d9289e38bbc57d00c0c082d4baeed1da00466

Comment (nowrep) [reply src/plugins/TabManager/tabmanagerwidget.cpp]: What's wrong with that commit?

Comment (srazi) [reply src/plugins/TabManager/tabmanagerwidget.cpp]: Before that commit actions of `if`s were different.
See this [part](https://github.com/QupZilla/qupzilla/commit/ea0d9289e38bbc57d00c0c082d4baeed1da00466#diff-76d20d37d43595d570512a1e29eeb8bbR474)

Comment (nowrep) [reply src/plugins/TabManager/tabmanagerwidget.cpp]: Yeah right, sorry 😆

**Pull request merged**